##the name of the output
Header header

bool is_contact

string body1_name
int8 body1_id

string body2_name
int8 body2_id

##the value to set output 

